module-api/src/main/java/org/example/dto/request/MoviesFilterRequestDto.java

```diff
@@ -0,0 +1,21 @@
+package org.example.dto.request;
+
+import jakarta.validation.constraints.Pattern;
+import jakarta.validation.constraints.Size;
+import lombok.Getter;
+import lombok.Setter;
+
+@Setter
+@Getter
+public class MoviesFilterRequestDto {
+    @Size(max = 255)
+    private String movieTitle;
+
+    @Pattern(
+            regexp = "|(?i)ACTION|ROMANCE|HORROR|SF",
+            message = "장르는 다음 중 하나여야 합니다: ACTION, ROMANCE, HORROR, SF"
+    )
+    private String genre;
```

Comment:
```suggestion
    @Pattern(
            regexp = "|(?i)ACTION|ROMANCE|HORROR|SF",
            message = "장르는 다음 중 하나여야 합니다: ACTION, ROMANCE, HORROR, SF"
    )
    private Genre genre;
```

Genre 타입으로 받을 순 없을까요? 스프링에선 해당 기능을 제공하니 아래 내용 확인해보세요

https://seongil-shin.github.io/posts/spring-controller%EC%97%90%EC%84%9C-enum%EC%9D%84-%EC%9D%B8%EC%9E%90%EB%A1%9C-%EB%B0%9B%EA%B8%B0/